Spanner/src/SpannerClient.php

```diff
@@ -252,11 +252,22 @@ public function __construct(array $config = [])
      * $batch = $spanner->batch('instance-id', 'database-id');
      * ```
      *
+     * Database role configured in the optional $options array
+     * will be applied to the session created by this object.
+     * ```
+     * $batch = $spanner->batch('instance-id', 'database-id', ['databaseRole' => 'Reader']);
+     * ```
+     *
      * @param string $instanceId The instance to connect to.
      * @param string $databaseId The database to connect to.
+     * @param array $options  [optional] {
+     *     Configuration options.
+     *
+     *     @type string $databaseRole The session owner database role.
```

Comment:
There's a scope to improve this comment

```suggestion
     *     @type string $databaseRole User created database role as specified in `CREATE ROLE` DDL statement.
```

Reply:
Something like this at other places too?

Reply:
Or maybe just use the one from proto: `The database role which created this session.`

Reply:
Changed like this
`The user created database role which creates the session.`